Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1291,21 +1294,17 @@ def resolve_detection_command():
     """
     args = demisto.args()
     ids = argToList(args.get('ids'))
-    usernames = argToList(args.get('username'))
-    if usernames and ids:
-        raise ValueError('Only one of the arguments ids or username should be provided, not both.')
-    if not usernames and not ids:
-        raise ValueError('One of the arguments ids or username must be provided, none given.')
-    if usernames:
-        ids = []
-        for username in usernames:
-            username_uuid = get_username_uuid(username)
-            ids.append(username_uuid)
+    username = args.get('username')
+    assigned_to_uuid = args.get('assigned_to_uuid')
+    comment = args.get('comment')
+    if username and assigned_to_uuid:
```

Comment:
what if neither of them was used?

Reply:
it's being handled in `resolve_detection` function:
<img width="475" alt="Screen Shot 2020-09-22 at 19 32 55" src="https://user-images.githubusercontent.com/21142167/93910927-7012c500-fd0a-11ea-8c80-da2dfab305d5.png">
I tested it as well.